src/modules/http/http.client.ts

```diff
@@ -0,0 +1,48 @@
+import { HttpService } from '@nestjs/axios';
+import { AxiosResponse } from 'axios';
+import { catchError, lastValueFrom, Observable, ObservableInput } from 'rxjs';
+
+import { RequestHeaders } from './type/headers.type';
+
+// TODO APIM-336: commonise with AcbsHttpService
+export class HttpClient {
```

Comment:
I'd like to refactor `AcbsHttpService` to use `HttpClient`. As it will involve changes to quite a few files, I want to do this in a separate PR